doc/source/whats_new.rst

```diff
@@ -9,6 +9,10 @@ Current
 Changelog
 ~~~~~~~~~
 
+   - Add support for single dipole fitting by `Eric Larson`_
+
+   - Add support for spherical BEM models by `Eric Larson`_
```

Comment:
It's still a boundary element model (what matters is the changes at the boundary borders), it just uses a spherical surface. Note e.g. Matti has a function called `mne_make_sphere_bem`...